open-metadata-resources/open-metadata-archives/open-metadata-types/src/main/java/org/odpi/openmetadata/opentypes/OpenMetadataTypesArchive3_9.java

```diff
@@ -0,0 +1,496 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.opentypes;
+
+
+import org.odpi.openmetadata.repositoryservices.archiveutilities.OMRSArchiveBuilder;
+import org.odpi.openmetadata.repositoryservices.archiveutilities.OMRSArchiveHelper;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.archivestore.properties.OpenMetadataArchive;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.archivestore.properties.OpenMetadataArchiveType;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.typedefs.TypeDefAttribute;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.typedefs.TypeDefAttributeStatus;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.typedefs.TypeDefPatch;
+import org.odpi.openmetadata.repositoryservices.ffdc.OMRSErrorCode;
+import org.odpi.openmetadata.repositoryservices.ffdc.exception.OMRSLogicErrorException;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * OpenMetadataTypesArchive builds an open metadata archive containing all the standard open metadata types.
+ * These types have hardcoded dates and guids so that however many times this archive is rebuilt, it will
+ * produce the same content.
+ * <p>
+ * Details of the open metadata types are documented on the wiki:
+ * <a href="https://egeria-project.org/types/">The Open Metadata Type System</a>
+ * </p>
+ * <p>
+ * There are 8 areas, each covering a different topic area of metadata.  The module breaks down the process of creating
+ * the models into the areas and then the individual models to simplify the maintenance of this class
+ * </p>
+ */
+public class OpenMetadataTypesArchive3_9
+{
+    /*
+     * This is the header information for the archive.
+     */
+    private static final String                  archiveGUID        = "bce3b0a0-662a-4f87-b8dc-844078a11a6e";
+    private static final String                  archiveName        = "Open Metadata Types";
+    private static final String                  archiveDescription = "Standard types for open metadata repositories.";
+    private static final OpenMetadataArchiveType archiveType        = OpenMetadataArchiveType.CONTENT_PACK;
+    private static final String                  archiveVersion     = "3.9";
+    private static final String                  originatorName     = "Egeria";
+    private static final String                  originatorLicense  = "Apache 2.0";
+    private static final Date                    creationDate       = new Date(1588261366992L);
```

Comment:
@lpalashevski should I ignore this one?

Reply:
I would say that is not critical in the context of how it is used in this exact place in the code mainly because we are not manipulating dates but only storing the information when the archive was updated. In future we might want to migrate java.util.Date to other recommended types such as java.time.Instant. I will mark it as "Not relevant" for now.

Other interesting observation is that we haven't updated this field value since archive v2.0 - maybe it is good practice to start doing this again so please put current timestamp for the new version 3.11

Reply:
ok

Reply:
updated